extension/js/tree.js

```diff
@@ -175,9 +175,16 @@ function sortCiteData(data) {
 			});
 			// for each previousSortedRowID (parentCiteIDs of the current rowSubset), let's populate subset in order
 			previousSortedRowIDs.forEach(function (parentCiteID) {
-				subset.push(...rowSubset.filter(function (cite) {
+				// find the cites that share a parentCiteID
+				unsortedSubset = rowSubset.filter(function (cite) {
 					return cite.parentCiteID === parentCiteID;
-				}));
+				});
+				// order these cites by score, highest to lowest going from left to right
+				sortedSubset = unsortedSubset.sort(function (a, b) {
+					return b.score - a.score;
```

Comment:
unlike citeID, we are now reliant on score being an int value.  But, they now sort on score!